src/handlers/modulesLoader.ts

```
@@ -53,17 +163,78 @@ export const loadModules = async (
       const { info, router } = module;
 
       if (info.version === airlinkVersion) {
-        logger.log(`Loading module: ${info.name} (v${info.moduleVersion})`);
+        // Determine module group
+        let moduleGroup = 'Other';
+
+        if (info.name.toLowerCase().includes('admin')) {
+          moduleGroup = 'Admin';
+        }
+        if (info.name.toLowerCase().includes('auth')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)